app/controllers/bookings_controller.rb

```diff
@@ -1,5 +1,11 @@
 class BookingsController < ApplicationController
   before_action :get_offer
+  skip_before_action :get_offer, only: :index
```

Comment:
before_action :get_offer, only: :new

Wouldn't this replace both lines below?

before_action :get_offer
 skip_before_action :get_offer, only: :index